web/template/clients.gotmpl

```diff
@@ -96,7 +99,8 @@
                         aria-sort="none">
                         <button
```

Comment:
what's the reason for not using a normal `<a href="">` link for all of these buttons? Feels like we're reinventing the anchor tag using javascript

Reply:
The arrows are added with CSS, so technically we can apply them to any HTML element

Reply:
But they dont work with an anchor tag I did try it :D